README.md

```diff
@@ -17,6 +17,16 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
 
 If you have suggestions or problems, please [open an issue](https://github.com/nextcloud/spreed/issues) or contribute directly :)
 
+
+### Installing
+
+There are several ways of installing Talk. If you just need to enable local access it's enough to just enable the app from the Nextcloud App Store. 
```

Comment:
Nextcloud Talk is really easy to install. You just need to enable the app from the [Nextcloud App Store](https://apps.nextcloud.com/apps/spreed) and everything will work out of the box.

Reply:
@Ivansss Everything won't work out of the box as you need Turn for it to work with video calls. But yeah sure, I'll change it.